### PR TITLE
Leave club

### PR DESCRIPTION
**Pull request by steven-solar**
This PR allows members to leave a club they are a member of, removing them from the member list and redirecting them to their home page.

### CODE REVIEW

File: capstone-frontend/src/components/ClubPage.jsx [hunk 1]
```diff
@@ -111,12 +111,25 @@ class ClubPage extends Component {
...
+  }
+
   componentDidMount() {
     this.fetchData();
   }
 
   render() {
-    const isOwner = this.state.owner && this.state.club.ownerID === window.localStorage.getItem('userID');
+    const userID = window.localStorage.getItem('userID');
+    const isOwner = this.state.owner && this.state.club.ownerID === userID;
```
Comment (antoniolinhart) [inline capstone-frontend/src/components/ClubPage.jsx:146]: I like separating this into 2 lines, it makes it look much cleaner and understandable

**Review by antoniolinhart — APPROVED** [review]
LGTM!

**Review by VivekShankar — APPROVED** [review]
LGTM

**Review by MauroGuerrero20 — APPROVED** [review]
LGTM

File: capstone-frontend/src/components/ClubPage.jsx (hunk 1 above)
Comment (steven-solar) [reply capstone-frontend/src/components/ClubPage.jsx:146]: Thanks!

**Pull request merged**